assets/src/dashboard/animations/constants.js

```diff
@@ -27,6 +27,10 @@ export const ANIMATION_TYPES = {
   ZOOM: 'zoom',
 };
 
+export const ANIMATION_EFFECTS = {
+  FADE_IN: 'effect-fade-in',
```

Comment:
To make sure animation effect names don't collide with existing animation parts, I figure we could prepend each name with `effect-`.